src/configuration/app-containers.md

```diff
@@ -9,19 +9,19 @@ You control your application and the way it will be built and deployed on
 Platform.sh via a single configuration file, `.platform.app.yaml`, located at
 the root of your application folder inside your Git repository.
 
-Here's an example of a minimalist `.platform.app.yaml` file for PHP:
+An example of a minimalist `.platform.app.yaml` file for PHP is below:
 
 ```yaml
 # .platform.app.yaml
 
-# The name of this app. Must be unique within a project.
+# The name of this application which must be unique within a project.
 name: 'app'
 
 # The type key specifies the language and version for your application.
 type: 'php:7.0'
 
-# On PHP, there are multiple build flavors available. Pretty much everyone except
-# Drupal 7 users will want the composer flavor.
+# On PHP, there are multiple build flavors available. Pretty much everyone
+# except Drupal 7 users will want the composer flavor.
```

Comment:
Since this line is a comment and most linters will balk when you go past 80 characters.